numpy/linalg/tests/test_linalg.py

```diff
@@ -1115,7 +1139,7 @@ class ArraySubclass(np.ndarray):
         assert_(isinstance(a, np.ndarray))
 
 
-class _TestNorm(object):
+class _TestNormGeneral(object):
 
     dt = None
     dec = None
```

Comment:
Repeating these definitions in both `_TestNormGeneral` and `_TestNorm2D` seems weird to me - can you put them in `_TestNormBase`?

Reply:
done.